packages/app/src/components/SearchBar/index.js

```diff
@@ -0,0 +1,102 @@
+import React, { useEffect, useState, useContext } from 'react';
+import { NominationsDataContext } from '../../utils/context/NominationsContext';
+import nominationsAPI from '../../utils/API/nominationsAPI';
+
+const SearchBar = () => {
+  const [searchTerm, setSearchTerm] = useState();
+  const [loading, setLoading] = useState(false);
+  const [SearchResultData, setSearchResultData] = useState([]);
+  const [errorMessageActive, setErrorMessageActive] = useState(false);
+  const [NominationsData, setNominationsData] = useContext(
+    NominationsDataContext
+  );
+  const [showErrorMessage, setShowErrorMessage] = useState(false);
+
+  useEffect(() => {
+    findAllNominations();
+  }, []);
+
+  function findAllNominations() {
+    nominationsAPI
+      .getNominations()
+      .then((res) => {
+        setNominationsData(res.data);
+      })
+      .catch((err) => console.log(err));
+  }
+
+  function findSearchResults(searchTerm) {
+    let results = [];
+    NominationsData.filter((nomination) => {
+      if (formatSearch(nomination.providerName) === formatSearch(searchTerm)) {
```

Comment:
you can do all three of these filters in one filter function ;) 

Give it a shot and then let a tech lead know if you need help